app/javascript/context/subreducers/ApplicationDetailsSubreducer.js

```diff
@@ -0,0 +1,148 @@
+import LEASE_STATES from 'context/actionCreators/application_details/leaseSectionStates'
```

Comment:
Another structure-related question: I'm wondering about the trade-off of having all of the reducers in one spot vs having them close to the rest of their relevant stuff. 

I think this is another point for ApplicationDetails being a bit vague too.